tools/orchestrator/src/download.ts

```diff
@@ -4,10 +4,11 @@ import { execSync } from 'child_process';
 import * as urlLib from 'url';
 import * as path from 'path';
 import * as mkdirp from 'mkdirp';
+import * as os from 'os';
 
 const DEFAULT_VERSION: string = '9.7.1.62043';
 const VERSIONS_URL: string = 'https://repox.jfrog.io/repox/api/search/versions?g=org.sonarsource.sonarqube&a=sonar-application&remote=0&repos=sonarsource-releases&v=*'
-const CACHE_PATH: string = path.join(__dirname, '..', 'test', 'cache');
+const CACHE_PATH: string = path.join(os.homedir(), '.sonar');
```

Comment:
I think declaring type `string` is a bit redundant here, it can be reliably inferred

Reply:
done